packages/next/src/server/route-modules/app-route/module.ts

```diff
@@ -143,6 +144,11 @@ export class AppRouteRouteModule extends RouteModule<
    */
   public readonly staticGenerationAsyncStorage = staticGenerationAsyncStorage
 
+  /**
+   * A reference to the static generation async storage.
```

Comment:
Looks like copy pasta'd from above. One nit about these comments is they don't really explain what they're used for, just what they are, which maybe would be worth expanding on here

Reply:
whoops, yeah, tbh, I just followed the pattern here but it's unclear to me why these are even instance members and not just imported from the graph. cc @wyattjoh do you know why we have this style of construction?